core/src/com/unciv/ui/multiplayer/FriendList.kt

```diff
@@ -0,0 +1,85 @@
+package com.unciv.ui.multiplayer
```

Comment:
We were having the "pure" separation of logic, metadata and UI lately... This might be better in metadata after all? maybe logic, there's some - no metadata. Then again, the data doesn't actually _live_ in an instance of this class, it's more of a utility... So logic?

As aside (I'm fine with the design) - some purists would say by the name this should be the actual container, and implement some Collection interface, then an instance of it would 'live' in settings. Just saying. Maybe a Kdoc for the class could clarify. If you want. The similarity to a Collection is still there, so some could argue the method names should follow patterns - add, remove, set. Your parameters and return types differ a little, so I'm not criticizing.